docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -11,11 +11,63 @@ Before consolidating a new QoS Policy, it is usually set using this generic QoS
 Consequently, this section is prone to frequent updates so the user is advised to check latest changes after upgrading
 to a different release version.
 
-``PushMode`` QoS Policy
-^^^^^^^^^^^^^^^^^^^^^^^
+.. _push_mode_qos_policy:
+
+``Push Mode`` QoS Policy
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Fast DDS DataWriters are enabled using the ``push mode``.
+This implies they will send new samples to matched readers immediately after adding them to their queue.
+For writers that produce non periodic bursts of data, this may imply saturating the network with a lot of packets,
+increasing the possibility of losing them on unreliable (i.e. UDP) transports.
+Depending on their QoS, DataReaders may also have to ignore some received samples, so they will have to be resent.
+
+Configuring the DataWriters on ``pull mode`` offers an alternative by letting each reader pace its own data stream.
+It works by notifying the reader what it is missing, and wait for it to request only as much as it can handle.
+At the cost of greater latency, this model can deliver reliability while using far fewer packets than in ``push mode``.
+
+DataWriters periodically announce the state of their queue by means of a heartbeat.
```

Comment:
I'd like a reference for the heartbeat

Reply:
Done on b31ded0e91ad900fbf5f775f4651eba187a34ec9